package.json

```diff
@@ -57,47 +62,51 @@
 		"validate-npm-package-name": "^6.0.0"
 	},
 	"devDependencies": {
-		"@eslint-community/eslint-plugin-eslint-comments": "^4.4.1",
-		"@eslint/js": "^9.18.0",
-		"@release-it/conventional-changelog": "^10.0.0",
+		"@eslint-community/eslint-plugin-eslint-comments": "4.4.1",
+		"@eslint/js": "9.22.0",
+		"@release-it/conventional-changelog": "10.0.0",
+		"@types/eslint-plugin-markdown": "2.0.2",
 		"@types/estree": "^1.0.6",
-		"@types/node": "^22.0.0",
+		"@types/node": "22.13.10",
 		"@types/semver": "^7.5.6",
 		"@types/sort-object-keys": "^1.1.3",
 		"@types/validate-npm-package-name": "^4.0.2",
-		"@vitest/coverage-v8": "^3.0.0",
-		"@vitest/eslint-plugin": "^1.1.25",
-		"console-fail-test": "^0.5.0",
-		"cspell": "^8.3.2",
-		"eslint": "^9.18.0",
+		"@vitest/coverage-v8": "3.0.9",
+		"@vitest/eslint-plugin": "1.1.38",
+		"console-fail-test": "0.5.0",
+		"create-typescript-app": "^2.26.1",
+		"cspell": "8.17.5",
+		"eslint": "9.22.0",
 		"eslint-doc-generator": "^2.1.2",
-		"eslint-plugin-eslint-plugin": "^6.4.0",
-		"eslint-plugin-jsdoc": "^50.6.1",
-		"eslint-plugin-jsonc": "^2.18.2",
-		"eslint-plugin-markdown": "^5.1.0",
-		"eslint-plugin-n": "^17.15.1",
-		"eslint-plugin-perfectionist": "^4.6.0",
-		"eslint-plugin-regexp": "^2.7.0",
-		"eslint-plugin-yml": "^1.16.0",
-		"husky": "^9.0.0",
+		"eslint-plugin-eslint-plugin": "6.4.0",
+		"eslint-plugin-jsdoc": "50.6.8",
+		"eslint-plugin-jsonc": "2.20.0",
+		"eslint-plugin-markdown": "5.1.0",
+		"eslint-plugin-n": "17.16.2",
+		"eslint-plugin-perfectionist": "4.10.1",
+		"eslint-plugin-regexp": "2.7.0",
+		"eslint-plugin-yml": "1.17.0",
+		"husky": "9.1.7",
 		"jsonc-eslint-parser": "^2.4.0",
```

Comment:
🐛 This is right to be here, but when I run `npx create-typescript-app` it gets removed. Hmm.